_tools/release_docs/main.go

```diff
@@ -59,6 +59,17 @@ func processFile(filename string) error {
 		return err
 	}
 
+	title := getTitle(fileBytes)
+	if len(title) == 0 {
+		title = filename
+		if splits := strings.SplitN(title, "/docs/", 2); len(splits) == 2 {
```

Comment:
This is really brittle.  something like docs/user-guide/examples/foo.md would break, as would examples/bigapp/docs/foo.md.

Given that we already know outputDir and the literals "docs" and "examples", could we modify the filepath.Walk() call to pass those into processFile and strip that suffix?

```
for _, dir := range allDocsDirs {  // allDocsDirs = [ "docs", "examples" ], fix copyFiles, too
    prefix := path.Join(*outputDir, dir)
    filepath.Walk(prefix, func(...) { processFile(path, prefix) }
```